mkdocs.yml

```
@@ -171,6 +171,7 @@ nav:
               - Create PR: docs/contribute/general/git/pr-create.md
               - PR review process: docs/contribute/general/git/review.md
           - Contribution Guidelines: docs/contribute/general/contrib-guidelines-gen.md
+          - Refinement Timeblock: docs/contribute/general/refinement-guide.md
```

i would call it refinement process
```suggestion
          - Refinement Process: docs/contribute/general/refinement-guide.md
```